src/ui.form-select/USelect.vue

```diff
@@ -688,10 +688,6 @@ const filteredOptions = computed(() => {
       )
     : [...props.options];
 
-  if (!normalizedSearch) {
-    return options.slice(0, props.optionsLimit || options.length);
```

Comment:
and how now we can limit the options length?

Reply:
The same way as before, it's done one line below :|